src/elements/emby-input/emby-input.css

```diff
@@ -21,6 +21,10 @@
         border: 0;
     }
 
+input:required { 
+box-shadow:none; 
```

Comment:
Please use the following as a selector, to be more precise in your change:

`.emby-input:required`

Please also respect the formatting of the surrounding code (indentation and spacing)